index.js

```diff
@@ -6,15 +6,27 @@ const app = express()
 app.use(bodyParser.json())
 app.use(cors())
 
- 
-const {initializeDBconnection} = require("./Database/DBconnect")
+
+const { initializeDBconnection } = require("./Database/DBconnect")
 initializeDBconnection()
 
-app.get("/", (req,res) => {
-  res.json({success: true, mesg: "Ya its working"})
+app.get("/", (req, res) => {
+  res.json({ success: true, mesg: "Ya its working" })
 })
 
+const userRouter = require("./Users/users.router.js")
+app.use("/", userRouter)
+
+const videosRouter = require("./Videos/videos.router.js")
+app.use("/videos", videosRouter)
+
+const videoLibraryRouter = require("./VideoLibrary/videoLibrary.router.js")
+app.use("/video-library", videoLibraryRouter)
```

Comment:
Here you can add that authVerify middleware (assuming "/video-library" is your only private route). 
app.use("/video-library",authVerify, videoLibraryRouter)